[#194] 업데이트 위한 conflict 해결

hyuns66 opened the pull request:
## #️⃣연관된 이슈

- #190 
- #188 

## 📝작업 내용

- 위 두 개 PR에서 발생한 conflict 수정했습니다.
- 어제 플레이스토어 업데이트 해야해서 제가 직접 pull 받아서 conflict 풀었어요

## PR 발행 전 체크 리스트

- [ ] 발행자 확인
- [ ] 프로젝트 설정 확인
- [ ] 라벨 확인
- [ ] 코드 린트 확인

## 스크린샷 (선택)

## 💬리뷰 요구사항(선택)

- 리뷰어가 특별히 봐주었으면 하는 부분이 있다면 작성해주세요
  - 아마 이 PR 합치면 위 두 PR은 닫아질텐데 @cyjadela @jieeeunnn conflict 해결하는 과정에서 이슈 없었는지 리뷰 한번씩만 해주세요!

cyjadela committed:
feat/#187 : 다른 앱 위에 표시 권한 팝업창 UI 구현

cyjadela committed:
feat/#187 : 경로 검색 화면에서 알림 등록 시 권한 팝업 로직 구현

cyjadela committed:
feat/#187 : 다른 앱 위에 권한 요청 Snackbar 구현

cyjadela committed:
feat/#187 : 상세 경로에서 다른 앱 위에 표시 권한 로직 구현

cyjadela committed:
chore/#187 : ktlint 적용

jieeeunnn committed:
fix/#180 : 홈 화면 status bar 패딩 적용

jieeeunnn committed:
fix/#189 : 출발지 검색 화면 status bar 패딩 적용

hyuns66 committed:
Merge branch 'fix/home-status-bar#189' of https://github.com/depromeet/16th-team6-Android into feat/alarm-fix#177

hyuns66 committed:
Merge branch 'feat/overlay#187' of https://github.com/depromeet/16th-team6-Android into feat/alarm-fix#177

hyuns66 committed:
chore : 배포 정보 변경

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/alarm-fix#177

hyuns66 committed:
feat/#177 : 알람 종료 로직 추가

- 기존에는 알람 종료하기를 눌러도 설정한 알람이 제대로 종료되지 않는 버그가 있었습니다.
- 정상종료되도록 수정했고 API 오류로 인해 정상 종료되지 않을경우 파이어베이스 로깅하도록 수정했습니다.

hyuns66 committed:
feat/#177 : 알람 제한시간 추가

- 2분 후 알람이 자동 종료되도록 수정했습니다.

cyjadela reviewed (approved):
확인했습니다 ~ ! 충돌 해결 아주 감사드리옵니다

hyuns66 committed:
feat/#177 : 알람 기능 Amplitude 로깅

- 유저가 알람화면에서 행동을 했는지
- 행동하는데까지 얼마나 걸렸는지
에 대한 데이터 로깅 코드 추가했습니다.

hyuns66 committed:
feat/#177 : 알람 2분뒤 종료시 노티 날리는 기능 추가

- 몇몇 NotificationID 수정했습니다.
- BaseForegroundServiceManager를 필요로 하지 않는 코드 삭제했습니ㅏ.

hyuns66 committed:
chore : linting
